db/migrate/20250214153835_move_blacklisted_events_to_settings_changes.rb

```diff
@@ -0,0 +1,85 @@
+    "ManageIQ::Providers::Ovirt::InfraManager"                       => ["UNASSIGNED", "USER_REMOVE_VG", "USER_REMOVE_VG_FAILED", "USER_VDC_LOGIN", "USER_VDC_LOGIN_FAILED", "USER_VDC_LOGOUT"],
+    "ManageIQ::Providers::Openstack::InfraManager"                   => ["identity.authenticate"],
+    "ManageIQ::Providers::Redhat::InfraManager"                      => ["UNASSIGNED", "USER_REMOVE_VG", "USER_REMOVE_VG_FAILED", "USER_VDC_LOGIN", "USER_VDC_LOGIN_FAILED", "USER_VDC_LOGOUT"],
+    "ManageIQ::Providers::OracleCloud::ContainerManager"             => ["com.oraclecloud.resourcequeryservice.searchresources", "com.oraclecloud.identitysignon.federatedinteractiveloginattempt", "serviceapi"],
+    "ManageIQ::Providers::Azure::ContainerManager"                   => ["storageAccounts_listKeys_BeginRequest", "storageAccounts_listKeys_EndRequest", "deployments_exportTemplate_BeginRequest", "deployments_exportTemplate_EndRequest"],
+    "ManageIQ::Providers::Amazon::ContainerManager"                  => ["ConfigurationSnapshotDeliveryCompleted", "ConfigurationSnapshotDeliveryStarted", "ConfigurationSnapshotDeliveryFailed"],
+    "ManageIQ::Providers::OracleCloud::CloudManager"                 => ["com.oraclecloud.resourcequeryservice.searchresources", "com.oraclecloud.identitysignon.federatedinteractiveloginattempt", "serviceapi"],
+    "ManageIQ::Providers::Openstack::CloudManager"                   => ["identity.authenticate", "scheduler.run_instance.start", "scheduler.run_instance.scheduled", "scheduler.run_instance.end"],
+    "ManageIQ::Providers::Azure::CloudManager"                       => ["storageAccounts_listKeys_BeginRequest", "storageAccounts_listKeys_EndRequest", "deployments_exportTemplate_BeginRequest", "deployments_exportTemplate_EndRequest"],
+    "ManageIQ::Providers::Amazon::CloudManager"                      => ["ConfigurationSnapshotDeliveryCompleted", "ConfigurationSnapshotDeliveryStarted", "ConfigurationSnapshotDeliveryFailed"],
+    "ManageIQ::Providers::IbmPowerVc::CloudManager"                  => ["identity.authenticate", "scheduler.run_instance.start", "scheduler.run_instance.scheduled", "scheduler.run_instance.end"],
+    "ManageIQ::Providers::IbmCic::CloudManager"                      => ["identity.authenticate", "scheduler.run_instance.start", "scheduler.run_instance.scheduled", "scheduler.run_instance.end"]
+  }.freeze
+
+  # TODO add the rest of the provider settings keys
+  # This is the mapping of BlacklistedEvent#provider_model -> SettingsChange#key
+  PROVIDER_MODEL_TO_SETTINGS_KEY = {
+    "ManageIQ::Providers::Vmware::InfraManager" => "/ems/ems_vmware/blacklisted_event_names"
+  }.freeze
+
+  SETTINGS_KEY_TO_PROVIDER_MODEL = {
+    "/ems/ems_vmware/blacklisted_event_names" => "ManageIQ::Providers::Vmware::InfraManager"
+  }.freeze
```

Comment:
Assuming the right and left are both unique

```suggestion
  SETTINGS_KEY_TO_PROVIDER_MODEL = PROVIDER_MODEL_TO_SETTINGS_KEY.invert
```

Reply:
I'm hoping so yeah, it uses the vendor not the provider and some providers override it so I need to prove that out